setup.py

```diff
@@ -614,6 +614,19 @@ def build_extensions(self):
                     os.makedirs(dst_dir)
                 self.copy_file(src, dst)
                 i += 1
+
```

Comment:
Just to check my understanding: after this PR, we will no longer have a standalone functorch build. Is that correct? (That isn't a problem or requirement, though it has some implications for the transition plan).

Users are still going to install functorch from source for a while (we have folks who run functorch in their CIs). For those people, we should make it so that installing functorch from the original location (https://github.com/pytorch/functorch) does something sane. Today there's a shipit sync from the functorch in Pytorch to https://github.com/pytorch/functorch to make it sane.

So here's the order I think we should do things in:
1. Turn off the shipit sync
2. Move functorch python code to a `torch._functorch` folder
3. Merge this PR
4. Change the pytorch/functorch repo to have a package that ends up doing `from torch._functorch import *`. If someone installs both PyTorch and `https://github.com/pytorch/functorch`, then `https://github.com/pytorch/functorch` will take precedence, but it will import the correct thing.

What do you think? Is there something easier we can do? If not, then I can handle 1, 2, and 4 (2 might take a week or so because we need to get some code reviews for it)

Reply:
@albanD suggested that instead of step (1), we can distribute a `_functorch_pytorch` and a `functorch` package with PyTorch. The `_functorch_pytorch` package would contain the extension, and the `functorch` package would just `from _functorch_pytorch import *`. This would avoid us needing to wait to finish code review for step 1.

Reply:
Nikita and I discussed more offline, we can just turn pytorch/functorch into a dummy package that doesn't contain anything. pytorch/functorch's setup.py lists `torch` as a dependency so that users will get `import functorch` automatically if they install pytorch/functorch from source (or if they install regular PyTorch)